scripts/scratch_ingest/loom_ingest.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import loompy
```

Comment:
Please add `loompy` and any other modules not in the Python standard library into [`requirements.txt`](https://github.com/broadinstitute/single_cell_portal/blob/master/requirements.txt).

Also, it's [helpful convention](https://www.python.org/dev/peps/pep-0008/#imports) to order standard modules (e.g. `time`, `os`) above third-party modules (e.g. `loompy`, `pyrebase`), with an empty line between standard and third-party modules group.